src/lib/all.js

```diff
@@ -15,4 +15,26 @@ export const all = (path, options = {}) => {
     .then(flatten)
 }
 
+export const allViaLoader = (loader, loaderOptions, apiOptions = {}) => {
```

Comment:
First this makes a call to get the total count.

Then, based on a default of 25 (or you can specify how to break it up), it fans out all the pages that should get you to the total.

Reply:
Could this have used the `totalViaLoader` loader?

Reply:
Yup, def!